icons/jar.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   version="1.1"
+   width="24"
+   height="24"
+   viewBox="0 0 24 24"
+   id="svg4"
+   sodipodi:docname="square-rounded.svg"
+   inkscape:version="1.1 (c68e22c387, 2021-05-23)"
```

Comment:
Lots of Inkscape cruft that needs cleaning out. I recommend a) SVGOMG or b) copy `java.svg` and only change the path `d` attribute.

Reply:
Thanks for suggesting SVGOMG, I didn't know about it existence. Now I know how to optimize SVG for future requests!